### DISCUSSION TITLE
AIR Visual C++ Redistributable

### DISCUSSION THREAD

**Discussion by cbrieger** [airsdk/Adobe-Runtime-Support#944] [Q&A] [answered]
I am trying to find any release notes or system requirements that state whether or not AIR versions 2.0, 2.5, 2.7, 3.1, 3.2, 3.7, 17.0, 22.0, 24.0, 31.0, 32.0, and 33.0 include or have a pre-requisite on Visual C++ Redistributable as well as the specific Visual C++ Redistributable versions for each. 

Does anyone know where I might be able to find this information? I'm not seeing any references to Visual C++ Redistributable in the release notes on Harman or Adobe's sites which leads me to believe that it is not included or a pre-req, but I'd appreciate any help in verifying this.

**Comment by ajwfrost** [comment, marked as answer]
I can't speak for sure for earlier versions, but the builds that we're doing are using the "Multithreaded" setting for the runtime library, i.e. statically linking this, rather than using "Multithreaded DLL" which would require the redistributable library to be present.

So we shouldn't have any dependency on those redistributables from 33.0 onwards at least; given that we just inherited the project files from Adobe, I would assume the same is true for 32 but it's possible that there may have been a different setting a long while ago..

**Reply by cbrieger** [reply]
Thank you! This is very helpful!
